backend/api/Services/MissionSchedulingService.cs

```diff
@@ -315,7 +322,14 @@ private async Task MoveInterruptedMissionsToQueue(IEnumerable<string> interrupte
                     Map = new MapMetadata()
                 };
 
-                await missionRunService.Create(newMissionRun);
+                try
+                {
+                    await missionRunService.Create(missionRun);
+                }
+                catch (UnsupportedRobotCapabilityException)
+                {
+                    logger.LogWarning($"Unsupported robot capability detected when restarting interrupted missions for robot {missionRun.Robot.Name}. This should not happen.");
```

Comment:
Not sure the service should be catching this? Can perhaps catch in higher levels?

Reply:
I would argue that we shouldn't handle it inside the Create function since then we will not be able to detect the specific cause of the error further down the stack. This is useful to report it to the user. The Create function also will not necessarily understand why it is that we do not expect this exception with this kind of mission run, since it doesn't treat Localisation and Return Home missions differently from normal missions which we do expect to potentially cause this exception.

Reply:
Yeah, good point